feature-group-definitions/read-write-pseudo-classes.yml

```diff
@@ -2,6 +2,15 @@ spec:
   - https://html.spec.whatwg.org/multipage/semantics-other.html#selector-read-only
   - https://drafts.csswg.org/selectors-4/#rw-pseudos
 caniuse: css-read-only-write
+status:
+  baseline: false
```

Comment:
This seems unlikely to be correct. Can you check on Firefox Android support?

Reply:
You're right. Opened https://github.com/mdn/browser-compat-data/pull/21453 to fix the underlying data.

What do we do with the status in the mean time? Drop it? Override it? Or just live with a "wrong" status until BCD releases and we refresh the status?